doc/roadmap_2023.md

```diff
@@ -0,0 +1,193 @@
+# Roadmap for 2023
+
+2022 was an interesting year for Vello. We made good progress on a number of fronts, solving some really tricky problems, and also did a rewrite into WGSL, but did not ship a usable renderer. Oh, and we renamed the project from piet-gpu as well.
```

Comment:
"solving some really tricky problems" could be more specific? Link to particularly interesting PRs or issue? 

Was wgsl tricky? - current phrasing suggests not.